addon/mixins/model-validator.js

```diff
@@ -248,10 +252,10 @@ export default Ember.Mixin.create({
     return validateThis;
   },
   _getCustomValidator: function(validation){
-    var customValidator = validation.custom;
-    if (Ember.typeOf(validation.custom) === 'object' && validation.custom.hasOwnProperty('validation')) {
-      customValidator = validation.custom.validation;
-    } 
+    var customValidator = validation;
+    if (Ember.typeOf(validation) === 'object' && validation.hasOwnProperty('validation')) {
+      customValidator = validation.validation;
+    }
```

Comment:
this just changes how the function is passed a validation object and returns the custom validation funciton (to account for it being iterated over in the function calling it)